bugrap/src/main/java/org/vaadin/bugrap/ui/LoginModel.java

```diff
@@ -15,7 +15,7 @@ public LoginModel(Navigator navigator) {
 	}
 	
 	public boolean login(String un, String pw) {
-		Reporter reporter = getRepository().authenticate(un, pw);
+		Reporter reporter = loginUser != null ? loginUser : getRepository().authenticate(un, pw);
```

Comment:
Is _loginUser_ a static variable? It shouldn't be, otherwise your app can only have one logged in user per JVM. There's a more suitable scope for storing the logged in user.

PS static variables should always be named like MY_VARIABLE..

Reply:
I know :) this is a workaround solution for now.
I will change this part next time

Reply:
Ok.